inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -166,7 +166,23 @@ void GNAModelSerial::Import(void *basePointer,
         InferenceEngine::OutputsDataMap& outputsDataMap) {
     is.exceptions(std::istream::failbit);
 
+    for (auto inputIndex = 0; inputIndex < modelHeader.nInputs; inputIndex++) {
+        uint32_t nameSize = 0;
+        readNBits<32>(nameSize, is);
+        std::vector<char> inName(nameSize);
+        readNBytes(inName.data(), nameSize, is);
+        inputNames.emplace_back(std::string(inName.begin(), inName.end() - 1));
```

Comment:
if you use std::string instead of std::vector, you will be able to push it directly.